templates/applications/app.html

```diff
@@ -11,12 +11,12 @@
     </div>
     <!-- Application -->
     <div class="col-md-9" style="min-height:400px">
-        {{ helper.render_app_title(app)}}
+        {{ helper.render_app_title(app, upload_method) }}
         <hr>
         {{ helper.render_overall_progress(app, n_tasks, overall_progress) }}
         {% if app.long_description %}
         <hr>
-            {{ app.long_description|safe}}
+            {{ app.long_description | e | markdown }}
```

Comment:
This is just a comment, not something we've to modify ;-) We should warn them that they can enforce markdown for their projects if they want. We keep it like this and disable it if they don't want it.